src/buildCommon.js

```diff
@@ -210,15 +239,15 @@ const makeOrd = function<NODETYPE: "spacing" | "mathord" | "textord">(
     } else if (fontOrFamily) {
         let fontName;
         let fontClasses;
-        if (fontOrFamily === "boldsymbol") {
-            const fontData = boldsymbol(text, mode, options, classes);
-            fontName = fontData.fontName;
-            fontClasses = [fontData.fontClass];
-        } else if (fontOrFamily === "mathit" ||
-                   utils.contains(mainitLetters, text)) {
-            const fontData = mathit(text, mode, options, classes);
+        if (fontOrFamily === "boldsymbol" || fontOrFamily === "mathnormal") {
+            const fontData = fontOrFamily === "boldsymbol"
+                ? boldsymbol(text, mode, options, classes)
+                : mathnormal(text, mode, options, classes);
             fontName = fontData.fontName;
             fontClasses = [fontData.fontClass];
+        } else if (utils.contains(mathitLetters, text)) {
+            fontName = "Main-Italic";
+            fontClasses = ["mathit"];
```

Comment:
This feels redundant.  Shouldn't the `mathdefault` call below take care of this case?